common/command.py

```diff
@@ -325,10 +325,10 @@ def get_obproxy_version(context):
 # Only applicable to the community version
 
 
-def get_observer_version_by_sql(ob_cluster, stdio=None):
+def get_observer_version_by_sql(context, ob_cluster, stdio=None):
```

Comment:
delete the input variable of stdio and replace it with context.stdio for output.

Reply:
Fixed it